docs/examples/extensions/simple-inbox-note/woocommerce-admin-simple-inbox-note-example.php

```diff
@@ -0,0 +1,29 @@
+<?php
+/**
+ * Plugin Name: WooCommerce Admin Simple Inbox Note Example
+ *
+ * @package WooCommerce\Admin
+ */
+
+// Import note class.
+require_once dirname( __FILE__ ) . '/class-simpleinboxnote.php';
+
+/**
+ * Add note.
+ */
+function add_note() {
+	SimpleInboxNote::possibly_add_note();
+}
+
+/**
+ * Delete note.
+ */
+function delete_note() {
+	SimpleInboxNote::possibly_delete_note();
+}
+
+// Attempt to add the note on plugin activation.
+add_action( 'activated_plugin', 'add_note' );
```

Comment:
I got a fatal error when activating the plugin unfortunately. It might be because in my local env, this plugin has loaded before Woo Admin does?

```
Fatal error: Trait 'Automattic\WooCommerce\Admin\Notes\NoteTraits' not found in /var/www/html/wp-content/plugins/simple-inbox-note/class-simpleinboxnote.php on line 16
```

Reply:
Good catch! I may need to add checks for it. I'm assuming you're using wp-env?

Reply:
Yep, wp-env. This is my config:

```
{
	"phpVersion": "7.4",
	"core": null,
	"plugins": [
		".",
		"https://downloads.wordpress.org/plugin/woocommerce.zip",
		"../simple-inbox-note"
	],
	"config": {
		"JETPACK_AUTOLOAD_DEV": true,
		"WP_DEBUG_LOG": true,
		"WP_DEBUG_DISPLAY": true,
		"ALTERNATE_WP_CRON": true
	}
}
```

Reply:
I've modified the script import to run on hook's callback instead, I've tested in my local wp-env and worked. Please give it a go!

Reply:
Nice, that works now!